python/rubin/rag/extra_scripts/scrape_github.py

```diff
@@ -117,38 +82,414 @@ def repos_in_org(org_name: str = "lsst-dm") -> list[str]:
         )
         repos.extend(res.json())
 
-    return [repo["full_name"] for repo in repos]
+    return [
+        repo["full_name"]
+        for repo in repos
+        if ("data" not in repo["name"].lower())
+        and ("dustmaps" not in repo["name"].lower())
+        and ("gen2" not in repo["name"].lower())
+        and ("legacy" not in repo["name"].lower())
```

Comment:
I would also add this. sorry I keep adding things
```suggestion
        and ("legacy" not in repo["name"].lower())
        and("test" not in repo["name"].lower())
```

Reply:
Thanks, I added this.